src/_includes/install/sampledata/sample-data-rc1-cli-24.md

```diff
@@ -0,0 +1,86 @@
+      ```
+
+   *  Example for {{site.data.var.ee}}:
+
+      ```bash
+      composer require magento/product-enterprise-edition {{ page.guide_version }}.0 --no-update
+      ```
+
+      ```bash
+      composer require magento/module-bundle-sample-data:100.4.* magento/module-widget-sample-data:100.4.* magento/module-theme-sample-data:100.4.* magento/module-catalog-sample-data:100.4.* magento/module-customer-sample-data:100.4.* magento/module-cms-sample-data:100.4.*  magento/module-catalog-rule-sample-data:100.4.* magento/module-sales-rule-sample-data:100.4.* magento/module-review-sample-data:100.4.* magento/module-tax-sample-data:100.4.* magento/module-sales-sample-data:100.4.* magento/module-grouped-product-sample-data:100.4.* magento/module-downloadable-sample-data:100.4.* magento/module-msrp-sample-data:100.4.* magento/module-configurable-sample-data:100.4.* magento/module-product-links-sample-data:100.4.* magento/module-wishlist-sample-data:100.4.* magento/module-swatches-sample-data:100.4.* magento/sample-data-media:100.4.* magento/module-offline-shipping-sample-data:100.4.* magento/module-gift-card-sample-data:100.4.* magento/module-customer-balance-sample-data:100.4.* magento/module-target-rule-sample-data:100.4.* magento/module-gift-registry-sample-data:100.4.* magento/module-multiple-wishlist-sample-data:100.4.* --no-update
+      ```
+
+   {% include note.html
+      type="info"
+    content="To upgrade to a Release Candidate, append `-rc<x>` to the version of each module. For example, `-rc3`."
+  %}
+
+1. Update dependencies:
+
+   ```bash
+   composer update
+   ```
+
+1. If prompted, enter your [authentication keys].
+1. Wait for dependencies to update.
+
+### Finish your upgrade
+
+After you have reset file system permissions:
+
+1. If you haven't done so already, log in to your Magento server as, or switch to, the Magento file system owner.
+1. Change to your Magento installation directory.
+1. Manually clear the `var/cache`, `var/page_cache`, and `var/generation` directories.
+
+   A sample command follows:
+
+   ```bash
+   rm -rf var/cache/* var/page_cache/* var/generation/*
+   ```
+
+1. Upgrade Magento:
+
+   ```bash
+   php bin/magento setup:upgrade
```

Comment:
I thought we were removing `php` from bin/magneto commands.

Reply:
hmm...won't let me comment on the cli-rc1-samp.md below. Do you need a space between %include on L8?

Reply:
I just broke the symlink and created a new file. I only modified lines 35 & 45 as mentioned in the PR description.